dojo/static/dojo/css/dojo.css

```diff
@@ -1031,7 +1031,7 @@ div.custom-search-form {
 
 @media (min-width: 320px) {
     #page-wrapper {
-        margin: 75px 0 0 0;
+        margin: 0 0 0 0;
```

Comment:
Why the change in margins ? I didn't run your code locally so I can't really make sense of that, but I thought it was only a css filter so maybe that's not required ?

Reply:
Update: I realized, it might be to apply the filter properly everywhere ? otherwise you had a white line ?